.builders/images/linux-x86_64/Dockerfile

```diff
@@ -46,9 +72,12 @@ ENV PATH="/opt/python/${PYTHON_VERSION}/bin:${PATH}"
 ENV LD_LIBRARY_PATH="/opt/python/${PYTHON_VERSION}/lib:${LD_LIBRARY_PATH}"
 
 # Set up virtual environment for Python 3
+# Note: virtualenv v20.22.0 dropped support for creating py2-based virtual environments,
```

Comment:
Let's do what the Windows image does and install this in each base Python so we don't have to track versions